src/components/message/message-content/message-content.style.ts

```diff
@@ -1,7 +1,8 @@
 import styled from "styled-components";
 
-const MessageContentStyle = styled.div`
-  padding: 15px 50px 15px 20px;
+const MessageContentStyle = styled.div<{ showCloseIcon?: boolean }>`
```

Comment:
It's not the end of the world because it's unlikely to ever change, but it's better to have one source of truth of your Typescript props. So, ideally you would import it from the base component and using Pick to grab the correct type. Happy to approve anyway because as I say, unlikely to ever change.